build.gradle

```diff
@@ -10,8 +10,8 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.3'
-
+        classpath 'com.android.tools.build:gradle:3.5.3'
```

Comment:
Can you PR a change to update our project to `3.5.3` as well so that we keep the tools versions in sync please? We're currently on `3.5.1` in `develop`.